setup.py

```diff
@@ -32,7 +32,8 @@
     install_requires=[
         "bravado-core >= 4.2.2",
         "crochet >= 1.4.0",
-        "fido >= 2.1.0, < 3.2.0",
+        "fido >= 2.1.0, <= 3.2.0",
```

Comment:
I originally added it when I was porting fido to python 3 because I knew it would break it. I'm fine with < 3.3 but I would keep the restriction as fido is rapidly growing and in a highly dynamic state